### ISSUE TITLE
Hyper parameter tuning for Metapath2vec model parameters

### ISSUE THREAD

**Issue by deepakagrawal** [deepakagrawal/Hack4Rare#2]
We need to tune the following hyperparameters in order to improve the embedding loss as well as training loss on sample nodes.

- embed-dim
- walk_length
- context_size
- walks_per_node
- num_negative_samples
